### PR TITLE
Lighting Tweaks

### PR DESCRIPTION
**Pull request by chefferz**
Tweaks lighting wattage to be far more realistic to futuristic tech.

1) Lighting now takes half the wattage as normal, A standard light tube draws only 80w now, vs 160w. 
2) The emergency lighting color is now fire engine/emergency red
3) Night shift lighting is now slightly brighter to ensure hallways aren't near pitch black in some areas.

### CODE REVIEW

**Comment by Poojawa** [comment]
The power draw is so that the engine has reason to be powered. This'll just make solar only stations easier

**Comment by chefferz** [comment]
> 
> 
> The power draw is so that the engine has reason to be powered. This'll just make solar only stations easier

The RP server already merged one that made it even lower than this. This game was designed in the mid 2000s before there was anything such as LED lighting/actual efficient lighting. It just boils down to, it's like 25XX and there's weapons that can launch BSAs across galaxies, yet we've got stupid inefficient lights? 

Right now a ballast bulb I can purchase at home depot takes 28w, and LED ones 20w. a 90W LED would be BRIGHT. like 9K Lumen bright.

At one point I'd like to add in some efficiency tweaks. Ideally I'd like to make janitors able to upgrade lights with a new light bulb that draws less power, as it would give them something to do, but I'm not experienced coding enough to do something to that extent.

**Comment by deathride58** [comment]
its retrofuturism. tbh i wouldn't be opposed to adding fluorescent light tubes/bulbs as an rnd upgrade, and keeping the current incandescent ones as-is. all the necessary groundwork to support em is already in place. give janitors and engineers something to do etc etc

**Comment by chefferz** [comment]
> 
> 
> its retrofuturism. tbh i wouldn't be opposed to adding fluorescent light tubes/bulbs as an rnd upgrade, and keeping the current incandescent ones as-is. all the necessary groundwork to support em is already in place. give janitors and engineers something to do etc etc

There's still a bit of pitch black areas on delta so 0.70 covers it just enough.

I'd really like a LITTLE bit more efficiency, so would 15w per lum sound a bit more fair?  (A normal bulb is 8 lum, so 120w)

Ideally if RnD got upgraded bulbs I'd want those stupid efficient, like +/- 50W

**Comment by nik707** [comment]
@deathride58 final comments?

File: code/modules/power/lighting.dm [hunk 1]
```diff
@@ -214,13 +214,13 @@
 	var/nightshift_enabled = FALSE	//Currently in night shift mode?
 	var/nightshift_allowed = TRUE	//Set to FALSE to never let this light get switched to night mode.
 	var/nightshift_brightness = 8
-	var/nightshift_light_power = 0.45
+	var/nightshift_light_power = 0.70
```
Comment (deathride58) [inline code/modules/power/lighting.dm]: This is way too close to the baseline light power (which is 0.75). With this proposed value, it'll be impossible to actually notice any difference between baseline lighting and night shift lighting, and there'd no longer be any mechanical advantages/disadvantages to night shifts (such as, but not limited to, nightmares getting some wiggle room with station lighting, podpeople needing to stay closer to station lights, botany plants growing slower with roundstart light setups, etc etc). This proposed value practically removes night shifts.

**Review by deathride58 — CHANGES_REQUESTED** [review]

**Comment by deathride58** [comment]
If the issue is that certain maps have dark spots, then it'd be better to make adjustments to the map's light layouts directly, as those dark spots are present regardless if night shifts are active or not.

**Comment by Linzolle** [comment]
To be perfectly honest I'd rather night shift lighting just completely removed, it's annoying at best and makes it impossible to see in some areas at worst.

**Comment by Tupinambis** [comment]
I have never met one player that likes night shift lighting. I'd rather it just be removed.

**Comment by Linzolle** [comment]
Night shift lighting doesn't even make sense IC, why would we need to stimulate the circadian rhythm of species that are currently supposed to be awake and working?

**Comment by Trilbyspaceclone** [comment]
If night shift was only on for low pop it would be better imo as it leaves slower draining APCs and such but being a Rng event that can happen at any time just is kinda bad and has no real logic to is

**Comment by silicons** [comment]
i honestly don't like the whole "change balance to be realistic" thing because it usually changes it for the worse.

**Comment by silicons** [comment]
stale and lack of maintainer interest

**Pull request closed without merging**